companies/1and1-gb.json

```diff
@@ -17,6 +17,7 @@
         "https://contact.1and1.co.uk/contact",
         "https://beta.companieshouse.gov.uk/company/03953678"
     ],
+    "custom-objection-template": "doesnt-exist",
```

Comment:
🚫 **[companylint]** <[custom-template-existence](https://github.com/datenanfragen/data/issues/1120)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record specifies `custom-objection-template` of `doesnt-exist` but that isn't available for `request-language` of `en`.